cluster/gce/gci/configure.sh

```diff
@@ -356,7 +356,7 @@ function try-load-docker-image {
   set +e
   local -r max_attempts=5
   local -i attempt_num=1
-  until timeout 30 ${LOAD_IMAGE_COMMAND:-docker load -i} "${img}"; do
+  until timeout 30 "${LOAD_IMAGE_COMMAND:-docker load -i}" "${img}"; do
```

Comment:
this line doesn't run in my local environment. it's same as mentioned https://github.com/kubernetes/kubernetes/pull/76694/files#diff-4233e27a9ed05c56fb788f03a08e245fL42
in order to solve this, converting `LOAD_IMAGE_COMMAND` to an array is good?